thrift/src/test/java/com/linecorp/armeria/client/thrift/ThriftOverHttpClientTServletIntegrationTest.java

```diff
@@ -191,20 +189,20 @@ public static void destroyServer() throws Exception {
         });
     }
 
-    @Before
-    public void setUp() {
+    @BeforeEach
+    void setUp() {
         SessionProtocolNegotiationCache.clear();
         sendConnectionClose.set(false);
     }
 
     @Test
-    public void sendHelloViaHttp1() throws Exception {
+    void sendHelloViaHttp1() throws Exception {
         final AtomicReference<SessionProtocol> sessionProtocol = new AtomicReference<>();
         final HelloService.Iface client = newSchemeCapturingClient(http1uri(HTTP), sessionProtocol);
 
         for (int i = 0; i <= MAX_RETRIES; i++) {
             try {
-                assertEquals("Hello, old world!", client.hello("old world"));
+                Assertions.assertEquals("Hello, old world!", client.hello("old world"));
```

Comment:
I think IntelliJ refactor might have un-static imported the assertions. Though is migrating to junit5 also the time to migrate to assertj? ;)

Reply:
Fixed :-)